packages/donor/src/screens/UpcommingDonation/UpcomingDonationScreen.tsx

```diff
@@ -61,11 +117,37 @@ export default function UpcomingDonationScreen({
 
               <div className={styles.detailLabel}>מתי?</div>
               <div className={styles.detailValue}>
-                {donationDate.toLocaleDateString(
+                {eventDateStart.toLocaleDateString(
                   "he-He",
                   DateUtils.ShortDateFormat
                 )}
               </div>
+              <div className={styles.link} onClick={handleClick}>
```

Comment:
Please extract the entire code of the `Add to Calendar` **button** to a dedicated component